File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.js
function solution(a, b) {
   let answer = 0;
        
    const a1 = a.toString();
    const b1 = b.toString();
    // a와 b를 문자열로 변환합니다.
        
    const sum1 = a1 + b1;
    const sum2 = b1 + a1;
    // a와 b를 붙입니다.
        
    const tmp1 = parseInt(sum1);
    const tmp2 = parseInt(sum2);
    // 붙인 문자열을 다시 정수로 변환합니다.
        
    if (tmp1 >= tmp2) {
        answer = tmp1;
    } 
    else {
        answer = tmp2;
    }
    // tmp1이 tmp2보다 크거나 같은 경우 tmp1을 반환하고, 그렇지 않은 경우 tmp2를 반환합니다.
    return answer;
}

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String a1 = Integer.toString(a);
		String b1 = Integer.toString(b);
		// int로 선언된 a,b를 String으로 변환한다
		
		String sum1 = a1 + b1;
		String sum2 = b1 + a1;
		// String으로 변환된 a,b를 붙인다
		
		int tmp1 = Integer.parseInt(sum1);
		int tmp2 = Integer.parseInt(sum2);
		// 순서를 바꾸어 붙인 sum1,sum2을 다시 int로 변환한다
		
        if (tmp1 >= tmp2) {
            answer = tmp1;
        }	// 만약 tmp1이 tmp2 보다 더 크거나 같다면 tmp1을 반환하고
        else {
            answer = tmp2;
        }	// 만약 tmp2가 tmp1 보다 더 크다면 tmp2을 반환한다
        return answer;
    }
}